macros/Compat.ejs

```diff
@@ -37,6 +37,7 @@ var output = '';
 var category = query.split(".")[0];
 var legendItems = new Set(); // entries will be unique
 
+// TODO: Migrate localization to Fluent ( https://www.npmjs.com/package/fluent )
```

Comment:
Please remove this TODO. I don't think we decided to use Fluent. A decision likely requires an ADR. https://github.com/mdn/mdn/tree/master/ADRs